src/components/InfoBanner/blocks/InfoBannerCard.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { ReactNode } from 'react';
+import styled from 'styled-components';
+
+import { BoxProps, Box } from '../../Box/Box';
+import { Link as WaveLink } from '../../Link/Link';
+import { Text } from '../../Text/Text';
+
+import { ICON_VARIANTS, IconBox, InfoBannerVariants, ROLE_VARIANTS, RoundedBox } from '../InfoBanner';
+
+interface CardProps extends BoxProps {
+    children: ReactNode;
+    variant?: InfoBannerVariants;
+    emphasized?: boolean;
+}
+
+const LinkList = styled.ul`
+    margin-block-start: 0.3rem;
+    margin-block-end: 0.3rem;
+    padding-inline-start: 1.5rem;
+
+    li {
+        line-height: 1.1;
+    }
+`;
+
+const Title = ({ children }: { children: string }) => (
```

Comment:
It looks like we always had an issue that the title `span`, but ideally, it should be flexible and maybe a headline by default. Created a bug #338

Reply:
I suppose we can change it to `<Headline>` in this MR